msteams-platform/build-your-first-app/build-and-run.md

```diff
@@ -1,81 +1,192 @@
 
-Sideload your app in Teams using the following steps:
+**To enable app sideloading in Teams**
 
-> [!NOTE]
-> To enable sideloading before you sideload your app in Teams, follow the steps in [Turn on app sideloading](../concepts/build-and-test/prepare-your-o365-tenant.md#enable-custom-teams-apps-and-turn-on-custom-app-uploading).
+1. Sign in to the [Microsoft 365 admin center](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/homepage#/) with your admin credentials.  
+1. Select **Show All** > **Teams**. 
 
-1. Select the **F5** key to launch a Teams web client in Visual Studio Code.
-1. To display your app content in Teams, specify that where your app is running (`localhost`) is trustworthy:
-   1. Open a new tab in the same browser window (Google Chrome by default) which opened after pressing **F5**.
-   1. Go to `https://localhost:3000/tab` and proceed to the page.
-1. Go back to Teams. In the dialog, select **Add for me** to install your app.
-:::image type="content" source="../assets/images/build-your-first-app/tab-running.png" alt-text="Screenshot showing an example 'Hello, World!' personal tab app running in Teams.":::
+   ![image of admin center menu](~/assets/images/prepare-test-tenant/admin-center.png)
 
-🎉 Congratulations! Your app is running in Teams.
+   > [!Note] 
+   > It can take up to 24 hours for the **Teams** option to appear. 
 
-## Next step
+1. Go to **Teams apps** > **Setup policies** > **Global** (Org-wide default).
 
-Expand on the personal tab you just created or build another type of Teams app.
+   ![turn on sideload view](~/assets/images/prepare-test-tenant/turn-on-sideload.png)
+
+1. Turn on the **upload custom apps** toggle.
+
+1. Select **Save** to save the changes.
+
+   Your test tenant now allows custom app sideloading.
+
+   > [!Note]
+   > Check for issues before sideloading your app using the validation feature in App Studio, which is included in the toolkit. Fix the errors to successfully sideload the app.
+
+
+### Sideload your app
+
+1. In Visual Studio Code, open the Teams Toolkit.
+1. Go to **App Studio**.  
+1. Select **Test and Distribute** > **Install**.
+
+   :::image type="content" source="../assets/images/build-your-first-app/vscode-teams-toolkit-appstudio.png" alt-text="Screenshot showing how to sideload your app to Teams client with the Visual Studio Code Teams Toolkit.":::
+
+**Alternatively**
+
+1. Select the **F5** key to open browser window to install. This will skip the installation process in the **App Studio** and lauch Teams in your browser.
+1. In the installation dialog, select **Add** to install your app to Teams.
+
+   :::image type="content" source="../assets/images/build-your-first-app/vscode-teams-toolkit-install.png" alt-text="Screenshot showing how to sideload your app to Teams client.":::
+
+   > [!Note]
+   > App Studio is also available as a stand-alone app for Teams client.
+
+### Troubleshoot App Studio
```

Comment:
```suggestion
### Troubleshoot sideloading issues
```
The current title is inaccurate. This is for troubleshooting sideloading/installation issues. It does not have anything to do with App Studio issues.